### PR TITLE
Migrate the llvm blog

### PR DESCRIPTION
**Pull request by Honggeiry**

### CODE REVIEW

File: src/content/blog/images/LLVM-IR-runtime-environment-management.png
Comment (GabrielaReyna) [inline src/content/blog/images/LLVM-IR-runtime-environment-management.png:1]: Could you please remove the white background of this image and leave transparent?

File: src/content/blog/images/process-of-compiling.png
Comment (GabrielaReyna) [inline src/content/blog/images/process-of-compiling.png:1]: Similar for this one, transparent background and the black lines can be inverted to white.

**Review by GabrielaReyna — DISMISSED** [review]
Same for the rest of the images in the blog post if possible (no background and invert colors so anything that is dark is visible in dark mode)

**Review by bates64 — APPROVED** [review]
LGTM besides Gaby's review

File: src/content/blog/reducing-webassembly-size-by-exploring-all-executions-in-LLVM.md [hunk 1]
```diff
@@ -0,0 +1,287 @@
+---
+title: "PartialExecuter: Reducing WebAssembly size by exploring all executions in LLVM"
+description: |
+  Discover the "Partial Executer," an LLVM pass that optimizes WebAssembly by removing unreachable code. This article explains the technology, challenges, and benefits, ideal for those versed in LLVM basics.
+pubDate: "2022-03-10"
+categories:
+  - "technical-blog"
```
Comment (bates64) [inline src/content/blog/reducing-webassembly-size-by-exploring-all-executions-in-LLVM.md:7]:
```suggestion
tags:
  - Cheerp
```

**Pull request merged**